.env-devel

```diff
@@ -59,7 +59,7 @@ CLUSTERS_KEEPER_EC2_INSTANCES_PREFIX=""
 CLUSTERS_KEEPER_LOGLEVEL=INFO
 CLUSTERS_KEEPER_MAX_MISSED_HEARTBEATS_BEFORE_CLUSTER_TERMINATION=5
 CLUSTERS_KEEPER_PRIMARY_EC2_INSTANCES=null
-CLUSTERS_KEEPER_TASK_INTERVAL=30
+CLUSTERS_KEEPER_TASK_INTERVAL=00:00:30
```

Comment:
Yes, https://git.speag.com/oSparc/osparc-ops-deployment-configuration/ is updated.

Reply:
We usually add those MR links  in the PR here
![image](https://github.com/user-attachments/assets/9a2d3d41-5392-49a2-aabd-1d06364e62eb)

Reply:
Yup, I have to find them... the files in ops-config were updated already.

Reply:
Already put on description. https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/1003